Bigtable/src/Connection/Grpc.php

```diff
@@ -269,6 +269,14 @@ public function setIamPolicy(array $args)
         throw new \BadMethodCallException('This method is not implemented yet');
     }
 
+    /**
+     * @param array $args
+     */
+    public function testIamPermissions(array $args)
```

Comment:
I already did this in #1129, so you can just remove it.

It would help if you could describe your PR meaningfully and stick to that only. 
"Bigtable Client functions" doesn't convey good meaning.
And IMHO, it makes us difficult to develop in parallel (and also difficult to review) if you keep adding things not directly related to your PR. :)